application/backend/src/main/resources/apps/bulma/components/block/block.html

```diff
@@ -0,0 +1,28 @@
+<!--/*
+  ~
+  ~     Copyright (C) 2022 Dynamic Solutions
+  ~
+  ~     Licensed under the Apache License, Version 2.0 (the "License");
+  ~     you may not use this file except in compliance with the License.
+  ~     You may obtain a copy of the License at
+  ~
+  ~         http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~     Unless required by applicable law or agreed to in writing, software
+  ~     distributed under the License is distributed on an "AS IS" BASIS,
+  ~     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~     See the License for the specific language governing permissions and
+  ~     limitations under the License.
+  ~
+*/-->
+
+<sly data-sly-use.model="pl.ds.bulma.components.models.BlockComponent">
```

Comment:
You can simplify this a little bit. 
Firstly, we don't have to use a model, because we don't get anything from it.
Secondly, you can change the order of elements here. You can start by checking if the resource has any children. If not then There will be a default placeholder presented in CMS.. It will allow you to skip this div with padding in edit mode.
```
<sly data-sly-test="${resource.hasChildren}">
  <div class="block">
      <sly data-sly-include="${'parsys.html' @ prependPath='/libs/wcm/foundation/components/parsys'}"></sly>
  </div>
</sly>
```

Reply:
Ah yeah, I had a feeling that it could be done more appropriately. Just pushed the changes, thanks for the feedback!